modules/management/zammad.nix

```diff
@@ -0,0 +1,59 @@
+{ config, pkgs, ... }:
+let
+  database_name = "zammad";
+  username = "zammad";
+  http_port = 3000;
+  ws_port = 3001;
+in
+{
+  sops.secrets.zammad_db_pass.owner = username;
+
+  services = {
+    postgresql = {
+      enable = true;
+      ensureDatabases = [ database_name ];
+      ensureUsers = [
+        {
+          name = username;
+          ensurePermissions = {
+            "DATABASE ${database_name}" = "ALL PRIVILEGES";
+          };
+        }
+      ];
+    };
+
+    zammad = {
+      enable = true;
+      package = pkgs.zammad;
+      port = http_port;
+      websocketPort = ws_port;
+      database = {
+        createLocally = false;
+        type = "PostgreSQL";
+        host = "/run/postgresql";
+        name = database_name;
+        user = username;
+        passwordFile = config.sops.secrets.zammad_db_pass.path;
+      };
+    };
+
+    nginx = {
+      enable = true;
+      recommendedProxySettings = true;
+      virtualHosts = {
+        "project.${config.deployment-dd-ix.domain}" = {
```

Comment:
maybe a bit to generic: suggestions: `kanban.dd-ix.net`or `tickets.dd-ix.net`

Reply:
wanted to keep the `tickets`-Subdomain free for a potential Pretix-installation. what about `management` or `orga` ?

Reply:
orga sounds good